website/static/js/pages/notifications-config-page.js

```diff
@@ -18,3 +18,18 @@ $.ajax({
     $notificationsMsg.addClass('text-danger');
     $notificationsMsg.text('Could not retrieve notification settings.');
 });
+
+//Fixes profile settings side menu to left column
+function fixAffixWidth() {
+    $('.affix, .affix-top, .affix-bottom').each(function (){
```

Comment:
Instead of repeating this function in multiple places, try putting it in a js module that can be required by the files that need it. That way, if you have to change this function you do not need to change it in multiple places.